README.md

```diff
@@ -53,6 +55,12 @@ Once you're done, tear down MySQL.
 ./docker/run-mysql.sh stop leonardo
 ```
 
+### Run Leonardo locally
+This will start leonardo and the database locally and shut them down when it exits. You should be able to access swagger at https://local.broadinstitue.org:30443, but you need to allow it to be unsecured. It will not have a dataproc environment associated with it however.
```

Comment:
Can you elaborate on the dataproc comment?

Reply:
i was mistaken. i removed that comment